.github/exemptions.md

```diff
@@ -0,0 +1,31 @@
+# Reasons for ignoring certain checks
+
+## Megalinter Disabled Linters
+
+- jscpy - CopyPaste scanning is disabled
+- JAVASCRIPT_STANDARD - We use eslint instead
+
+## Specific linter exemptions
+
+- cspell - "ignoreRegExpList": "/[0-9a-f]{8}.js/gi" - scripts/downlaod.js will save files with this name
+- cspell - "ignoreRegExpList": "/[0-9a-f]{8}\\_ABI.json/gi" - scripts/downlaod.js will save files with this name
+- cspell - "ignoreRegExpList": "/0x[0-9a-f]{40}/gi" - ignore ETH addresses
+- cspell - "ignoreRegExpList": "/Qm[0-9a-zA-Z]{44}/g" - ignore IPFS CIDv0 addresses
+- cspell - "ignoreRegExpList": "/bafy[0-9a-zA-Z]{40}[0-9a-zA-Z]+/g" - ignore IPFS CIDv1 addresses
```

Comment:
See comments about IPFS CIDv1 format from comments here:
https://github.com/arbitraryexecution/automation-testing/pull/92

Reply:
addressed in e1250bd6ccf32489638864a367c380f8b73589cc